src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigAlignmentsConfigPicker.java

```diff
@@ -701,49 +701,31 @@ static GoodAndBadMappings splitGapsAndKeepChildrenTogether(final GoodAndBadMappi
     @VisibleForTesting
     static GoodAndBadMappings splitGapsAndDropAlignmentContainedByOtherOnRead(final GoodAndBadMappings configuration) {
         final List<AlignmentInterval> originalGoodMappings = configuration.goodMappings;
-        final List<Tuple2<Boolean, AlignmentInterval>> alignmentSplitChildren = originalGoodMappings.stream()
-                .flatMap(alignment -> {
-                    final Iterable<AlignmentInterval> split;
-                    if (alignment.containsGapOfEqualOrLargerSize(GAPPED_ALIGNMENT_BREAK_DEFAULT_SENSITIVITY)) {
-                        split = ContigAlignmentsModifier.splitGappedAlignment(alignment, GAPPED_ALIGNMENT_BREAK_DEFAULT_SENSITIVITY,
-                                SvCigarUtils.getUnclippedReadLength(alignment.cigarAlong5to3DirectionOfContig));
-                    } else {
-                        split = Collections.singletonList(alignment);
-                    }
-                    return Utils.stream(split).map(ai -> new Tuple2<>(true, ai));
-                }).collect(Collectors.toList());
-        final int count = alignmentSplitChildren.size();
+        final List<AlignmentInterval> gapSplit = new ArrayList<>(originalGoodMappings.size());
```

Comment:
Minor suggestion but I think formulating this as `stream` -> `flatMap` -> "collection as list" might be a little cleaner than creating the list upfront and then adding things to it inside `forEach`es.